Update clean-metadata-store.md (#13131)
apache/druid: docs/operations/clean-metadata-store.md

### Indexer task logs

> You can configure the Overlord to delete indexer task log metadata and the indexer task logs from local disk or from cloud storage.  The cleanup includes the `druid_tasks` and `druid_tasklogs` tables in the metadata database, and the task logs in deep storage.  (Note that `druid_tasklogs` is no longer used and will already be empty, unless the druid version is older.) 

Reviewer: ```suggestion
You can configure the Overlord to delete indexer task log metadata and the indexer task logs from local disk or from cloud storage.
During cleanup, the Overlord removes the `druid_tasks` and `druid_tasklogs` tables from [metadata storage](../configuration/index.md#metadata-storage) and removes task logs from deep storage.
Note that `druid_tasklogs` is no longer used and will already be empty, unless the druid version is older.
```

Reviewer: Two follow up questions:
1. Do you know what older version uses `druid_tasklogs`? Wondering if we can say something like "unless the Druid version is XX or older."
2. Should the table be called `druid_taskLog`? That's what's listed in the configuration table: https://druid.apache.org/docs/latest/configuration/index.html#metadata-storage

Reviewer: @paul-rogers The cleanup is based on a schedule and is configured by a few different properties. I updated my suggestion based on your recommendation. The properties to configure the cleanup are already in this section in the lines below (https://druid.apache.org/docs/latest/operations/clean-metadata-store.html#indexer-task-logs)

Reviewer 2: Looks good. The following is a bit unclear:

> (Note that `druid_tasklogs` is no longer used and will already be empty, unless the druid version is older.)

Maybe:

> The cleanup includes the `druid_tasks` table in the metadata database, and the task logs in deep storage. In previous versions of Druid task logs were stored in the the metadata database table `druid_tasklogs`. However, this table is no longer used in the current version and the table is always empty.

Also, I like your "periodically" wording so we know that that this isn't one shot:

> You can configure the Overlord to delete indexer logs periodically. Indexer logs include ...

Reviewer: Updated.

### Indexer task logs

You can configure the Overlord to periodically delete indexer task logs and associated metadata. During cleanup, the Overlord removes the following:

> * Indexer task logs from local disk or cloud storage.

Reviewer: ```suggestion
* Indexer task logs from deep storage.
```

* Indexer task log metadata from the `druid_tasks` and `druid_tasklogs` tables in [metadata storage](../configuration/index.md#metadata-storage). Druid no longer uses the `druid_tasklogs` table, and the table is always empty.

> Set these properties in the `overlord/runtime.properties` file.

Reviewer: ```suggestion

To configure cleanup of task logs by the Overlord, set the following properties in the `overlord/runtime.properties` file.
```

> * Indexer task log metadata from the `druid_tasks` and `druid_tasklogs` tables in [metadata storage](../configuration/index.md#metadata-storage). Previous versions of Druid stored task logs in the metadata database table `druid_tasklogs`. However, this table is no longer used in the current version, and the table is always empty.

Reviewer: ```suggestion
* Indexer task log metadata from the `druid_tasks` and `druid_tasklogs` tables in [metadata storage](../configuration/index.md#metadata-storage). Druid no longer uses the `druid_tasklogs` table, and the table is always empty.
```

> * Indexer task log metadata from the `druid_tasks` and `druid_taskLog` tables in [metadata storage](../configuration/index.md#metadata-storage). Previous versions of Druid stored task logs in the metadata database table `druid_tasklogs`. However, this table is no longer used in the current version, and the table is always empty.

Reviewer: ```suggestion
* Indexer task log metadata from the `druid_tasks` and `druid_tasklogs` tables in [metadata storage](../configuration/index.md#metadata-storage). Previous versions of Druid stored task logs in the metadata database table `druid_tasklogs`. However, this table is no longer used in the current version, and the table is always empty.
```
